misc/test/go.mod

```diff
@@ -3,13 +3,18 @@ module github.com/pulumi/examples
 go 1.16
 
 require (
-	github.com/aws/aws-sdk-go v1.37.31
+	github.com/aws/aws-sdk-go v1.38.35
 	github.com/mitchellh/go-homedir v1.1.0
+	github.com/pulumi/pulumi-trace-tool v0.0.0-20210611225312-ca1114c56a53
 	github.com/pulumi/pulumi/pkg/v3 v3.0.0
-	github.com/pulumi/pulumi/sdk/v3 v3.0.0
+	github.com/pulumi/pulumi/sdk/v3 v3.3.1
 	github.com/stretchr/testify v1.6.1
 	gopkg.in/airbrake/gobrake.v2 v2.0.9 // indirect
 	gopkg.in/gemnasium/logrus-airbrake-hook.v2 v2.1.2 // indirect
 )
 
 replace github.com/Azure/go-autorest => github.com/Azure/go-autorest v14.2.0+incompatible
+
+replace github.com/pulumi/pulumi/pkg/v3 => github.com/pulumi/pulumi/pkg/v3 v3.4.1-0.20210611165853-02c33000552c
```

Comment:
why are you replacing these? Can't we just upgrade the normal deps rather than having to manage these with replacements?

Reply:
Ah no good reason on using replace that's true. I was hesitating updating deps for all tests but if we can do it that's great too. For now I moved it aside so only bench tests run against pre-rel deps.